src/main/python/cfn_sphere/stack_configuration/__init__.py

```diff
@@ -9,22 +9,20 @@
 class Config(object):
```

Comment:
Just and idea, but the most versatile Configuration object I have devised so far is here:

  https://github.com/ImmobilienScout24/cbas/blob/master/src/main/python/cbas/configuration.py#L27